unittests/dataproviders/status.dataprovider.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Dataprovider;
+
+class Status {
+    public static function getStatusStringsProvider()
+    {
+        return [
+            'Good Service on all lines, non-peak' => [
+                new \DateTimeImmutable('Monday 13:00'),
+                '',
+                self::getGoodServiceData(),
+            ],
+            'Good service on all lines, peak' => [
+                new \DateTimeImmutable('Monday 9:00'),
+                '9:00 — Good service on all lines',
+                self::getGoodServiceData(),
+            ],
+            'Minor delays, non-peak' => [
+                new \DateTimeImmutable('Monday 13:00'),
+                "Minor delays on East Didsbury line\nMinor delays on Rochdale via Oldham line\nGood service on all other lines",
+                self::getMinorDelaysData(),
+            ],
+            'Minor delays, peak' => [
+                new \DateTimeImmutable('Monday 17:00'),
+                "17:00 — Minor delays on East Didsbury line\nMinor delays on Rochdale via Oldham line\nGood service on all other lines",
+                self::getMinorDelaysData(),
+            ],
+        ];
+    }
+
+    private static function getMinorDelaysData()
+    {
+        return json_decode('{
```

Comment:
I was thinking whether it would make sense to have these JSON-Strings extracted into a JSON file and do a `return json_decode(file_Get_contents(__DIR__ . '/_assets/minorDelayData.json'));` here…

The data can be easier edited and linted within a JSON-file than within a PHP-File.